ru/basics.html

```diff
@@ -230,16 +236,18 @@ <h2 id="functions">Функции</h2>
     arg.capitalize
   }
 }
+
+scala&gt; capitalizeAll("rarity", "applejack")
+res2: Seq[String] = ArrayBuffer(Rarity, Applejack)
 </pre>
 <h2 id="class">Классы</h2>
+Здесь мы объявили класс Calculator:
```

Comment:
I don't see this in the English version; therefore, I think it's best to remove it or add it to the English version as well.